pkg/client/status_reporter.go

```diff
@@ -27,10 +27,12 @@ import (
 )
 
 const (
-	unavailableMessage          string = "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
-	asExpectedReason            string = "AsExpected"
-	StorageNotConfiguredMessage        = "Prometheus is running without persistent storage which can lead to data loss during upgrades and cluster disruptions. Please refer to the official documentation to see how to configure storage for Prometheus: https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html"
-	StorageNotConfiguredReason         = "PrometheusDataPersistenceNotConfigured"
+	unavailableMessage                         string = "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
+	asExpectedReason                           string = "AsExpected"
+	StorageNotConfiguredMessage                       = "Prometheus is running without persistent storage which can lead to data loss during upgrades and cluster disruptions. Please refer to the official documentation to see how to configure storage for Prometheus: https://docs.openshift.com/container-platform/4.8/monitoring/configuring-the-monitoring-stack.html"
+	StorageNotConfiguredReason                        = "PrometheusDataPersistenceNotConfigured"
+	UserAlermanagerConfigMissconfiguredMessage        = "AlertmanagerConfig resources for user-defined alerting are enabled in the openshift-monitoring/cluster-monitoring-config configmap with the 'enableUserAlertmanagerConfig: true' field but a dedicated Alertmanager instance in openshift-user-workload-monitoring/user-workload-monitoring-config is also enabled which takes precedence. Please remove the 'enableUserAlertmanagerConfig' field in openshift-monitoring/cluster-monitoring-config and check if 'enableAlertmanagerConfig' is set to true in openshift-user-workload-monitoring/user-workload-monitoring-config."
+	UserAlermanagerConfigMissconfiguredReason         = "UserAlermanagerConfigMissconfigured"
```

Comment:
```suggestion
	UserAlertmanagerConfigMisconfiguredReason         = "UserAlertmanagerMisconfigured"
```
2 Changes 
- typo
- Doesn't `UserAlertmanagerMisconfigured` sound better?